src/esp/assets/ResourceManager.h

```diff
@@ -25,6 +28,7 @@
 #include "GltfMeshData.h"
 #include "MeshData.h"
 #include "MeshMetaData.h"
+#include "esp/gfx/configure.h"
```

Comment:
Why was this configure.h added?  The assets lib likely isn't getting linked against the gfx lib, so the include path for this file may not be being added.